mobile/src/foss/java/org/openhab/habdroid/ui/MapViewHelper.kt

```diff
@@ -74,9 +74,7 @@ object MapViewHelper {
                             return true
                         }
 
-                        override fun longPressHelper(p: GeoPoint): Boolean {
-                            return false
-                        }
+                        override fun longPressHelper(p: GeoPoint): Boolean = false
```

Comment:
When shortening this that way, one can additionally omit the return type declaration, I think: `override fun long press helper(p: GeoPoint) = false`

(Same multiple times below)